### DISCUSSION TITLE
Структурирование cache-файлов на гугл диске

### DISCUSSION THREAD

**Discussion by Bondian** [Chimildic/goofy#134] [Помощь с алгоритмом] [answered]
Вопрос не по алгоритму, но всё же...
Существует ли возможность записывать cache-файлы в новые папки внутри уже существующей "Goofy Data"?

**Comment by Chimildic** [comment, marked as answer]
С текущей версией кода Cache, нет

**Reply by Bondian** [reply]
Ясно. Спасибо.
